schema/auth.graphqls

```diff
@@ -16,6 +16,7 @@ enum Privilege {
   VEHICLE_VIN_CREDENTIAL
   VEHICLE_APPROXIMATE_LOCATION
   MANUFACTURER_DEVICE_LAST_SEEN
+  ATTESTATION_CREDENTIAL
```

Comment:
Can you remind me how we map these to the numbers? Like what maps `VEHICLE_NON_LOCATION_DATA` to `1`?

Reply:
Okay, here: https://github.com/DIMO-Network/telemetry-api/blob/c7f6f44583ef569623feb8b1bd1ec60c0f1cff7b/internal/auth/auth.go#L28 I think we need to do something here